static/js/click_handlers.js

```diff
@@ -536,6 +536,7 @@ exports.initialize = function () {
         ".user-presence-link, .user_sidebar_entry .user_circle, .user_sidebar_entry .selectable_sidebar_block",
         (e) => {
             e.stopPropagation();
+            gear_menu.close_if_needed();
```

Comment:
I don't think showing a tooltip would be expected to hide a menu -- can you explain your thinking here?

Reply:
This is the case when the user presses on any <code>streams</code> in the left sidebar . 
<strong>Image</strong>
![Screenshot from 2020-09-24 12-33-27](https://user-images.githubusercontent.com/53977614/94111852-3b198600-fe62-11ea-9740-440d79cda56b.png)

Isn't it better to close the gear menu when the user presses on any of the steam ?

Reply:
Shouldn't the gear menu close when the user browses through the user's list as the gear menu overlaps with the user's tab , hence hiding its visibility . ( The user must have an idea of which user is he pressing upon)